test/fixtures/compile-test-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Ah, no, it is used for that. It's run entirely manually. Perhaps I need to expose it in package.json ;-)